pkg/procinfo/procinfo.go

```diff
@@ -162,14 +186,20 @@ func NewProcessInfo() (*ProcInfo, error) {
 		return nil, err
 	}
 
-	// Iterate over each pid
+	// iterate over each pid
 	for _, procEntry := range entries {
 		pid, err := strconv.ParseUint(procEntry, 10, 32)
 		if err != nil {
 			continue
 		}
 		taskDir, err := os.Open(fmt.Sprintf("/proc/%d/task", pid))
+		if err != nil {
+			return nil, err
```

Comment:
better use continue.
It might be that some task exited while looping procfs. We don't want to fail tracee in such a case.

Reply:
no need to log. Just continue